alerting/provider/gchat/gchat.go

```diff
@@ -0,0 +1,139 @@
+package gchat
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/TwiN/gatus/v3/alerting/alert"
+	"github.com/TwiN/gatus/v3/client"
+	"github.com/TwiN/gatus/v3/core"
+)
+
+// AlertProvider is the configuration necessary for sending an alert using Google chat
+type AlertProvider struct {
+	WebhookURL string `yaml:"webhook-url"`
+	// Url to your gatus instance
+	GatusHost string `yaml:"gatus-url,omitempty"`
```

Comment:
Could you remove this for now, and create another issue to linking to the dashboard on alerts as feature? 

I feel like having the hostname as a variable somewhere could benefit other areas in Gatus, and I'd rather avoid adding something that will end up being a breaking change.

In the mean time, you can always use the alert description to link to the instance, even if it's not optimal.